package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "payback",
+  "version": "1.0.0",
+  "description": "The RESTful server dedicated to serve the PayUTC mobile app",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha --timeout 10000",
+    "lint": "eslint .. La date de parution",
```

Comment:
Is it a real command?

Reply:
npm init does not like when you specify that you want to implement eslint without installing it, it is changed